Sources/MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -47,6 +53,49 @@ extension Bundle {
         return true
     }
     
+    /**
+     Returns a dictionary of `MBXInfo.plist` in Mapbox Core Navigation.
+     */
+    static var mapboxCoreNavigationMBX: Dictionary<String, Any>? {
+        guard let fileURL = Bundle.mapboxCoreNavigation.url(forResource: "MBXInfo", withExtension: "plist"),
+              let infoDictionary = NSDictionary(contentsOf: fileURL) as? Dictionary<String, Any> else { return nil }
+        return infoDictionary
+    }
```

Comment:
This still fetches and parses the plist every time it’s called. This would make it compute only once, lazily on the first use:

```suggestion
    static let mapboxCoreNavigationInfoDictionary: [String: Any]? = {
        guard let fileURL = Bundle.mapboxCoreNavigation.url(forResource: "MBXInfo", withExtension: "plist"),
              let infoDictionary = NSDictionary(contentsOf: fileURL) as? Dictionary<String, Any> else { return nil }
        return infoDictionary
    }()
```